evergreen/config.yml

```diff
@@ -936,16 +969,20 @@ tasks:
 
 - name: baas-network-tests
   tags: [ "for_nightly_tests" ]
+  allowed_requesters: [ "ad_hoc", "patch", "github_pr" ]
+  # The network tests can take a really long time, so we give the test up to 4
+  # hours to complete
   exec_timeout_secs: 14400
   commands:
   - func: "launch remote baas"
     vars:
-      baas_branch: master
+      baas_branch: 3f31617aacfe5d31b9057fc298b735b60acd6424
       baas_proxy: On
   - func: "compile"
     vars:
       target_to_build: ObjectStoreTests
   - func: "wait for baas to start"
+  - func: "setup proxy parameters"
```

Comment:
don't we actually need to run this for the new tasks?

Reply:
Yes - this is what configures the proxy toxics for the tests and runs a shell script. This is run locally on the "test" host vm (not the remote baas) and configures the proxy via the tunnel for the Proxy HTTP Config port (8474).

Reply:
What I meant is that doesn't the new tasks need to invoke it as well?

Reply:
What confused me was that `network_tests` is not in this changes (was added previously). `network_tests` runs `baas-network-tests` which invokes `"setup proxy parameters"`